docs/testing/debug-with-vs-extension.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Visual Studio Code Debugger Extension for Office Add-ins
+description: 'Use a VS Code extension to debug your Office add-in.'
+ms.date: 05/14/2020
+localization_priority: Normal
+---
+
+# Visual Studio Code Debugger Extension for Office Add-ins
+
+The Visual Studio Debugger Extension for Office Add-ins allows you to use F5 in VS Code to debug your Office Add-in against the Edge runtime. This debugging mode is dynamic, allowing you to set breakpoints while code is running. You can see changes in your code immediately while the debugger is attached, all without losing your debugging session. Your code changes also persist, so you can see the results of multiple changes to your code. The following image shows this extension in action.
```

Comment:
Question about "code changes also persist": Does this mean your mid-debugging edits are saved? Would users expect this to not be the case? Maybe it's just the phrasing, but the line comes across as marketing the save feature, so I'm not sure it's needed (of course, this is dependent on other dynamic debugger behavior).